### PR TITLE
changed upgrade flow to better fit kuberentes

### PR DESCRIPTION
**Pull request by dannyzaken**
### Explain the changes
1. run upgrade_manager in `noobaa_init.sh` to perform the upgrade if needed
2. rewrote `upgrade_manager`:
   - scan an upgrade scripts directory and load all scripts according to old and new versions.
   - run all required script in order

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1.

### CODE REVIEW

File: frontend/src/app/reducers/system-reducer.js [hunk 1]
```diff
@@ -133,10 +133,11 @@ function onFailCollectSystemDiagnostics(state) {
 function _mapUpgrade(payload) {
     const { last_upgrade } = payload.upgrade;
 
+    // DZDZ - hardcoded initiator - not relevant anymore
```
Comment (nimrod-becker) [inline frontend/src/app/reducers/system-reducer.js]:
```suggestion
  
```

File: src/server/system_services/system_server.js [hunk 2]
```diff
@@ -504,9 +509,8 @@ function read_system(req) {
         //     dbg.log0('read_system: n2n_config.stun_servers', n2n_config.stun_servers);
         // }
 
-        let last_upgrade = system.last_upgrade && {
-            timestamp: system.last_upgrade.timestamp,
-            last_initiator_email: system.last_upgrade.initiator
+        let last_upgrade = system.upgrade_history.successful_upgrades[0] && {
+            timestamp: system.upgrade_history.successful_upgrades[0].timestamp
```
Comment (nimrod-becker) [inline src/server/system_services/system_server.js]: We should probably remove from the FE the initiator as well or we'll fail on schema

File: src/server/system_services/system_server.js [hunk 3]
```diff
@@ -578,10 +582,7 @@ function read_system(req) {
             system_cap: system_cap,
             has_ssl_cert: has_ssl_cert,
             cluster: cluster_info,
-            upgrade: {
-                last_upgrade: last_upgrade,
-                can_upload_upgrade_package: _get_upgrade_availability_status(cluster_info)
-            },
+            upgrade: { last_upgrade },
```
Comment (nimrod-becker) [inline src/server/system_services/system_server.js]: remove  can_upload_upgrade_package from FE as well

**Review by nimrod-becker — APPROVED** [review]

File: src/server/system_services/system_server.js (hunk 2 above)
Comment (dannyzaken) [reply src/server/system_services/system_server.js]: for now, I tried to reduce any changes in the FE. I filled in any fields required by the UI to not break it

**Pull request merged**